examples/inference/benchmark_llama.py

```diff
@@ -117,6 +133,8 @@ def benchmark_inference(args):
                 max_output_len=args.output_len,
                 prefill_ratio=1.2,
                 block_size=32,
+                tp_size=args.tp_size,
+                pp_size=args.pp_size,
```

Comment:
Do we need pp at present?

Reply:
No need for now. Just keep the original configs/args.

Reply:
Unused args removed.